scss/_alert.scss

```diff
@@ -54,18 +54,13 @@
 
 
 // scss-docs-start alert-modifiers
-// Generate contextual modifier classes for colorizing the alert.
-
+// Generate contextual modifier classes for colorizing the alert
 @each $state, $value in $theme-colors {
-  $alert-background: shift-color($value, $alert-bg-scale);
-  $alert-border: shift-color($value, $alert-border-scale);
-  $alert-color: shift-color($value, $alert-color-scale);
-
-  @if (contrast-ratio($alert-background, $alert-color) < $min-contrast-ratio) {
-    $alert-color: mix($value, color-contrast($alert-background), abs($alert-color-scale));
-  }
   .alert-#{$state} {
-    @include alert-variant($alert-background, $alert-border, $alert-color);
+    --#{$prefix}alert-color: var(--#{$prefix}#{$state}-text);
```

Comment:
Background colors haven't changed but text colors changed in this PR.
Considering the first `alert-primary` as an example:
- Before - Background (`#CFE2FF`) - Foreground (`#084298`): Normal Text was WCAG AAA
- Now - Background (`#CFE2FF`) - Foreground (`#0A57CA`): Normal Text is now only WCAG AA ⚠️ 

_Source: https://webaim.org/resources/contrastchecker/_

IDK if it is possible (haven't checked all the new color system) but I think it would make sense to me that a "text primary" on a "bg primary subtle" could be usable and WCAG AAA.

Reply:
i'd say aiming for AA is the best we can hope. AAA is very restrictive

Reply:
though sure, if we can nudge it to AAA all the better

Reply:
Yeah I think we're good right _now_, but I could see us darkening another shade if needed after alpha/beta release feedback. All color combos here in light and dark modes are minimum AA.